testing.rst

```diff
@@ -815,6 +815,92 @@ You can remove an existing field, e.g. a tag::
 .. index::
    pair: Tests; Configuration
 
+Testing file upload using CollectionType
+........................................
+
+By default, a CollectionType isn't available in the form during functional tests 
+if no data are passed by default, sometimes, you need to upload multiple files (using this CollectionType), 
+keeping in mind this logic, it can be hard to handle the upload.
+
+Let's imagine a simple form which define a collection of FileType::
+
+    <?php
+
+    namespace App\Form;
+
+    use App\Entity\Entity;
```

Comment:
`Entity\Entity` - I think it's bag example. You can use `User` or `Task` or another entity used in other documentation articles.

Reply:
What about `App\Entity\Album` with a `pictures` property?